styles/variables-silicon.less

```diff
@@ -155,7 +186,7 @@
 
 // RadioItem
 // ---------------------------------------
-@agate-radioitem-icon-border-width: 1px;
+@agate-radioitem-icon-border-width: 2px;
```

Comment:
If `2px` is a recurring design trend, we may need to solve this in one of several ways:
* Update the design to better support multi-resolution planning (use multiples of 3 for every value)
* Update the screenTypes.json file to more accurately reflect the final product's target display resolutions, rather than broad generalized screen resolutions, assigning a safer `pxPerRem` value with cleaner division values.
* Use a clever bit of trickery to get a clean `2px` value, putting it into a LESS variable, and using it all over this file: `~"calc(" 3px ~"- 1px)"` - This safely converts `3px` down to rem, then uses the browser to subtract `1px` for a clean `2px` end result. (The `~"..."` is LESS escaping, so LESS doesn't process the math before the conversion.)

Ex:
```css
@silicon-two-px: ~"calc(" 3px ~"- 1px)";

@agate-input-border-width: @silicon-two-px;
@agate-radioitem-icon-border-width: @silicon-two-px;
```